.eslintrc.yaml

```diff
@@ -117,18 +124,19 @@ rules:
   }, {
     selector: "ThrowStatement > CallExpression[callee.name=/Error$/]",
     message: "Use new keyword when throwing an Error."
-    }, {
-    selector: "CallExpression[callee.object.name='assert'][callee.property.name='fail'][arguments.length=1]",
```

Comment:
I took the rule out because it was part of a merge conflict when cherry-picking, and in those cases, I err on the side of more lenient linting in previous releases. I think you're right, and I should restore it.